src/api/server/authn.rst

```diff
@@ -345,3 +345,124 @@ headers to CouchDB with related requests:
 
 Note that you don't need to request :ref:`session <api/auth/session>`
 to be authenticated by this method if all required HTTP headers are provided.
+
+.. _api/auth/jwt:
+
+JWT Authentication
+====================
+
+.. note::
+    To use this authentication method make sure that the
+    ``{chttpd_auth, jwt_authentication_handler}`` value is added to the
+    list of the active :config:option:`chttpd/authentication_handlers`:
+
+    .. code-block:: ini
+
+        [chttpd]
+        authentication_handlers = {chttpd_auth, cookie_authentication_handler}, {chttpd_auth, jwt_authentication_handler}, {chttpd_auth, default_authentication_handler}
+
+`JWT authentication` enables CouchDB to use externally generated JWT tokens
+instead of defining users or roles in the ``_users`` database.
+
+The JWT authentication handler requires that all JWT tokens are signed by a key that
+CouchDB has been configured to trust (there is no support for JWT's "NONE" algorithm).
+
+Additionally, CouchDB can be configured to reject JWT tokens that are missing a
+configurable set of claims (e.g, a CouchDB administrator could insist on the ``exp`` claim).
+
+All claims presented in a JWT token are validated if presented, regardless of whether they
+are required.
+
+Two sections of config exist to configure JWT authentication;
+
+.. code-block:: ini
+
+    ;[jwt_auth]
+    ; List of claims to validate
+    ; required_claims =
+
+The `required_claims` config setting is a comma-separated list of additional mandatory
+JWT claims that must be present in any presented JWT token. A ``400 Bad Request`` is sent
```

Comment:
Please use the `:code 400: Invalid request` syntax so this is correctly formatted and hyperlinked. (or is it really Bad Request? everywhere else in the docs we use Invalid Request.)

Reply:
Bad Request is the usual text for a 400, yes (c.f, https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#400)